pkg/gui/controllers/remote_branches_controller.go

```diff
@@ -121,6 +127,14 @@ func (self *RemoteBranchesController) rebase(selectedBranch *models.RemoteBranch
 	return self.c.Helpers().MergeAndRebase.RebaseOntoRef(selectedBranch.FullName())
 }
 
+func (self *RemoteBranchesController) createSortMenu(selectedBranch *models.RemoteBranch) error {
+	return self.c.Helpers().Refs.CreateSortOrderMenu(func(sortOrder string) {
+		self.c.GetAppState().RemoteBranchSortOrder = sortOrder
+		self.c.SaveAppStateAndLogError()
+		self.c.Contexts().RemoteBranches.SetSelectedLineIdx(0)
```

Comment:
Nitpick: I'd do this only if the sort order has actually changed; otherwise we'd unnecessarily do a refresh and jump back to the top even though nothing changed.

Reply:
Agreed, should not need a refresh if state hasn't changed.